### PR TITLE
Create PEBCAC

### PR DESCRIPTION
**Pull request by h0us3cat**
## How Has This Been Tested?
#help

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by h0us3cat** [comment]
@Alderon86 
http://www.urbandictionary.com/define.php?term=Pebcac

**Comment by cptmedic** [comment]
But there is also http://www.urbandictionary.com/define.php?term=pebkac

We need to look at cross-system compatibility.

Also @bluemode you may need to review this for zoom zoom quality

File: docs/common-issues/PEBCAC [hunk 1]
```diff
@@ -0,0 +1,2 @@
+Problem Exists Between Chair And Computer.
+Used by technical support helpdesk staff to indicate that the problem with a user’s computer or experience is due to user error.
```
Comment (bluemode) [inline docs/common-issues/PEBCAC]: Please add support for Windows, Mac, and others as some will get confused if you do not specify.

Comment (bluemode) [inline docs/common-issues/PEBCAC]: Also no title was given to this document please add line # PEBCAC to first line ;)

**Comment by tjn182** [comment]
Great PR! Was able to identify and remove problem - now map goes zoom!

**Comment by sebastienvercammen** [comment]
The original is indeed `pebkac`, i.e. user error, ID-10T error, PICNIC, IBM: https://en.wikipedia.org/wiki/User_error

Jokes aside, PR closed, this is not going in. Sorry!

**Pull request closed without merging**